llvm/test/CodeGen/AMDGPU/insert-singleuse-vdst.mir

```diff
@@ -1,6 +1,5 @@
 # NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
 # RUN: llc -mtriple=amdgcn -mcpu=gfx1150 -mattr=+wavefrontsize32,-wavefrontsize64 -verify-machineinstrs -run-pass=amdgpu-insert-single-use-vdst %s -o - | FileCheck %s
```

Comment:
If you don't care about the wave size in this test, remove the -mattr option.

Reply:
Done